src/swapd/temporal_safety.rs

```diff
@@ -29,27 +29,21 @@ pub struct TemporalSafety {
     pub punish_timelock: BlockHeight,
     /// The minimum number of blocks that should remain unmined before the next transaction to be
     /// considered safe
-    pub race_thr: BlockSpan,
+    pub safety: BlockSpan,
```

Comment:
Update the comment here by copying the one from the config.

Reply:
Done c1adc5a7e4085e73e9ac0d459565d460ed83d53e